packages/config/css/animations.css

```
@@ -0,0 +1,298 @@
+@keyframes slideUpNormal {
+  0% {
+    height: inherit;
+    opacity: 1;
+  }
+  100% {
+    height: 0;
+    opacity: 0;
+  }
+}
+
+@keyframes panelSlideLeftOut {
```

Stylelint (17.11.0)

[error] Expected keyframe name "panelSlideLeftOut" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)